### PR TITLE
#248 removing stackstac from requirements into environment.yml as it'…

### PR DESCRIPTION
**Pull request by BnJam**
…s available in conda-forge

Moved `stackstac` from requirements to environment.yml. 

When `stackstac` is in requirements.txt, it is installed via Pip and the conda-forge installed `rasterio-1.2.8` is eliminated in place of PyPi `rasterio==1.2.6`. The PyPi installation does not have HDF4 format support. Now `stackstac-0.2.1` is available on conda-forge, the requirements.txt is no longer needed.

### CODE REVIEW

**Comment by scottyhq** [comment]
thanks @BnJam ! the `packages.txt` summaries get added automatically by CI, so for future PRs you actually don't need to include those

**Comment by scottyhq** [comment]
/condalock

re-running ...

File: pangeo-notebook/packages.txt [hunk 1]
```diff
@@ -135,22 +135,22 @@ freeglut-3.2.1
...
 geocube-0.0.17
 geographiclib-1.52
 geopandas-0.9.0
 geopandas-base-0.9.0
 geopy-2.2.0
 geos-3.9.1
-geotiff-1.6.0
-geoviews-core-1.9.1
+geotiff-1.7.0
+geoviews-core-1.8.2
```
Comment (scottyhq) [inline pangeo-notebook/packages.txt:153]: not sure why `geoviews-core` dropped from 1.9.1 to 1.8.2... also not sure if we need to keep using `geoviews-core` or just geoviews...

**Review by scottyhq — APPROVED** [review]

**Comment by BnJam** [comment]
> thanks @BnJam ! the `packages.txt` summaries get added automatically by CI, so for future PRs you actually don't need to include those

Thanks @scottyhq ! 

I saw that the packages.txt isn't being used by any of the Dockerfiles, but I was uncertain about not explicitly calling a version number from the environmental.yml file.

**Pull request merged**